.github/pull_request_template.md

```diff
@@ -13,6 +13,11 @@ Please provide a detailed description of what was done in this PR
 
 Please complete this section if any breaking changes have been made, otherwise delete it
 
+# Nodes audience
+
+Do the features included by this PR must be enabled only for a subset of nodes?
+In case, please specify how you handled it (e.g. by adding a flag with a default value...)
+
```

Comment:
Can we add this as an entry in `# Changes` section? I feel it is easier to check against the list than looking into each section.

Reply:
Yeah, good idea.
Will also raise again for heimdall, as it's already merged there.